source/PlayState.hx

```diff
@@ -2,15 +2,50 @@ package;
 
 import flixel.FlxSprite;
 import flixel.FlxState;
+import flixel.text.FlxText;
+import flixel.FlxG;
 
 class PlayState extends FlxState
 {
-	var hero:Hero;
+
+	var screensaverHero1:ScreensaverHero;
+	var screensaverHero2:ScreensaverHero;
+	var screensaverHero3:ScreensaverHero;
+	var screensaverHero4:ScreensaverHero;
+	var controllableHero:ControllableHero;
+
 	override public function create()
 	{	
-		hero = new Hero(300,300);
-		add(hero);
+
+		controllableHero = new ControllableHero(500, 400);
+		add(controllableHero);
+
+		screensaverHero1 = new ScreensaverHero(50, 0);
+		screensaverHero2 = new ScreensaverHero(50, 75);
+		screensaverHero3 = new ScreensaverHero(50, 150);
+		screensaverHero4 = new ScreensaverHero(50, 325);
+		screensaverHero1.velocity.x = FlxG.random.float(-75,75);
+		screensaverHero1.velocity.y = FlxG.random.float(-75,75);
+		screensaverHero2.velocity.x = FlxG.random.float(-75,75);
+		screensaverHero2.velocity.y = FlxG.random.float(-75,75);
+		screensaverHero3.velocity.x = FlxG.random.float(-75,75);
+		screensaverHero3.velocity.y = FlxG.random.float(-75,75);
+		screensaverHero4.velocity.x = FlxG.random.float(-75,75);
+		screensaverHero4.velocity.y = FlxG.random.float(-75,75);
```

Comment:
This is an effective way to randomize their directions, but the fact we had to repeat it so many times is a pretty good warning sign that we can find a better way.

In particular, since these lines are all about setting particular properties in our `ScreensaverHero`-type objects and we always do it exactly once after we create them, this looks like a really good candidate for something we could put inside `ScreensaverHero's constructor`.

That'll cut down on repeated lines of code and help the `ScreensaverHero` objects work out-of-the-box in the way we want: as things that float around randomly to make a beautiful screensaver.

Let's implement this change before we approve and merge this PR.